versions.yaml

```diff
@@ -189,7 +189,7 @@ externals:
     description: |
       OCI-based Kubernetes Container Runtime Interface implementation
     url: "https://github.com/cri-o/cri-o"
-    version: "v1.16.0"
+    version: "release-1.17"
```

Comment:
`release-1.17` is a branch, not a tag. afaik some branches are still under development (can accept new patches), therefore we won't have a 100% reproducible build/testing

cc @chavafg @GabyCT

Reply:
hmm, not sure what would be the best option here. In the past we have used the commit ID of the fix, In this case: [1].  
Then when new release is out, we can again update.

[1] https://github.com/cri-o/cri-o/commit/44c9932a4ab9c3836cb748eb9cd7736037d798eb

Reply:
@chavafg Using a commit id instead of a branch makes sense here.

Reply:
Hi @chavafg @devimc @amshinde 
I've changed to use a commit id here. In the future, it may be included in the v1.17.1. ;).

Reply:
thanks @Pennyzct 

> I've changed to use a commit id here. In the future, it may be included in the v1.17.1.

Could you file an issue for that?

Reply:
okay, raise the issue asap. ;).

Reply:
Done. See https://github.com/kata-containers/runtime/issues/2486. ;)